static/styles/lib/bootstrap/scss/utilities/_text.scss

```diff
@@ -8,6 +8,11 @@
 .text-nowrap         { white-space: nowrap !important; }
 .text-truncate       { @include text-truncate; }
 
+// alert datetime aligment
+.alert-datetime-wrapper {
```

Comment:
this css file looks pretty generic, do we have a specific css file to move this class?